.github/workflows/build_deploy.yml

```diff
@@ -211,3 +211,25 @@ jobs:
           # due to a duplicate wheel being present which will ensure that the rest
           # of the wheels will be uploaded if some are uploaded manually.
           skip_existing: true
+
+  build-and-publish-init-image:
+    # We have to wait for the PyPI job since the image that we publish depends on installing
```

Comment:
we also have the wheels here too if you want to use the local ones for building the image.

Reply:
this is out of scope of this change

Reply:
yup, just a thought I had